### PR TITLE
follow docs structure

### PR DESCRIPTION
**Pull request by DmitrySvetlichny**

### CODE REVIEW

File: Sources/Controllers/Help/MenuHelpDataService.swift [hunk 1]
```diff
@@ -213,44 +197,28 @@ final class MenuHelpDataService: NSObject, XMLParserDelegate {
...
-    
-    private func addArticleNode(_ node: ArticleNode, url: String) {
-        let parts = url.replacingOccurrences(of: kOsmAndUserBaseURL, with: "").split(separator: "/").map(String.init)
-        var currentNode = rootNode
-        for part in parts {
-            if let childNode = currentNode.childArticles.first(where: { $0.title == part }) {
-                currentNode = childNode
+    private func addArticleNode(_ node: ArticleNode) {
+        var parentNode: ArticleNode = rootNode
+        while true {
```
Comment (tigrim) [inline Sources/Controllers/Help/MenuHelpDataService.swift:202]: ?

File: Sources/Controllers/Help/MenuHelpDataService.swift [hunk 2]
```diff
@@ -108,16 +114,51 @@ final class MenuHelpDataService: NSObject, XMLParserDelegate {
...
-                        completion(chats as NSArray, nil)
+                    if let telegramChatsData = jsonDict["ios"] as? [String: Any],
+                       let chats = telegramChatsData["telegramChats"] as? [String: String] {
+                        var chatsArray: [TelegramChat] = []
+                        for (title, urlString) in chats {
+                            let chat = TelegramChat(title: title, url: urlString)
+                            chatsArray.append(chat)
+                        }
+                        
+                        telegramChats = chatsArray
```
Comment (tigrim) [inline Sources/Controllers/Help/MenuHelpDataService.swift:139]: `telegramChats = chats.map { TelegramChat(title: $0.key, url: $0.value") } ` ?

File: Sources/Controllers/Help/MenuHelpDataService.swift [hunk 3]
```diff
@@ -108,16 +114,51 @@ final class MenuHelpDataService: NSObject, XMLParserDelegate {
...
-                        completion(articles as NSArray, nil)
+                    if let popularArticlesData = jsonDict["ios"] as? [String: Any],
+                       let articles = popularArticlesData["popularArticles"] as? [String: String] {
+                        var articlesArray: [PopularArticle] = []
+                        for (title, urlString) in articles {
+                            let article = PopularArticle(title: title, url: self.urlPrefix + urlString)
+                            articlesArray.append(article)
+                        }
+                        
+                        popularArticles = articlesArray
```
Comment (tigrim) [inline Sources/Controllers/Help/MenuHelpDataService.swift:125]: comment down

File: Sources/Constants/OALinks.h [hunk 4]
```diff
@@ -40,7 +40,7 @@ static NSString * const kTroubleshootingSetup = @"https://docs.osmand.net/docs/u
 static NSString * const kTroubleshootingMap = @"https://docs.osmand.net/docs/user/troubleshooting/maps-data/";
 static NSString * const kTroubleshootingNavigation = @"https://docs.osmand.net/docs/user/troubleshooting/navigation/";
 static NSString * const kTroubleshootingTrackRecording =  @"https://docs.osmand.net/docs/user/troubleshooting/track-recording-issues/";
-static NSString * const kDocsBaseURL = @"https://docs.osmand.net/docs/user/";
+static NSString * const kDocsUserBaseURL = @"https://osmand.net/docs/user/";
```
Comment (tigrim) [inline Sources/Constants/OALinks.h:43]: Is this a duplicate of with kOsmAndUserBaseURL?

Comment (tigrim) [reply Sources/Constants/OALinks.h:43]: add `troubleshooting/` for url

**Review by tigrim — APPROVED** [review]

File: Sources/Controllers/Help/HelpDetailsViewController.swift [hunk 5]
```diff
@@ -106,8 +106,12 @@ final class HelpDetailsViewController: OABaseNavbarViewController {
         tableData.clearAllData()
         let childArticlesSection = tableData.createNewSection()
         for article in childArticles {
+            if article.title.contains("Android") {
```
Comment (alex-osm) [inline Sources/Controllers/Help/HelpDetailsViewController.swift:109]: Case insensitive?

**Pull request merged**